docs/introduction/en/references.md

```diff
@@ -0,0 +1,83 @@
+---
+id: conflux_refs
+title: References
+custom_edit_url: https://github.com/Conflux-Chain/conflux-developer-site/edit/master/docs/introduction/en/references.md
+keywords:
+  - conflux
+  - references
+---
+
+## Scan
+* mainnet: https://confluxscan.io/
+* testnet: https://testnet.confluxscan.io/
+
+## chainId
+* mainnet: 1029
+* testnet: 1
+
+## networkId
+* mainnet: 1029
+* testnet: 1
+
+## CIP37 address prefix
+* mainnet: `cfx`
+* testnet: `cfxtest`
+
+## CIP37 address type
+* user
+* contract
+* builtin
+* null
+
+## Internal contract addresses
+
+### hex address
+* AdminControl: 0x0888000000000000000000000000000000000000
+* SponsorWhitelistControl: 0x0888000000000000000000000000000000000001
+* Staking: 0x0888000000000000000000000000000000000002
+
+### CIP37 mainnet address
+* AdminControl: cfx:aaejuaaaaaaaaaaaaaaaaaaaaaaaaaaaaa2mhjju8k
+* SponsorWhitelistControl: cfx:aaejuaaaaaaaaaaaaaaaaaaaaaaaaaaaaegg2r16ar
+* Staking: cfx:aaejuaaaaaaaaaaaaaaaaaaaaaaaaaaaajrwuc9jnb
+
+### CIP37 testnet address
+* AdminControl: cfxtest:aaejuaaaaaaaaaaaaaaaaaaaaaaaaaaaaawby2s44d
+* SponsorWhitelistControl: cfxtest:aaejuaaaaaaaaaaaaaaaaaaaaaaaaaaaaeprn7v0eh
+* Staking: cfxtest:aaejuaaaaaaaaaaaaaaaaaaaaaaaaaaaajh3dw3ctn
+
+## 1820 address
+* hex address: 0x88887eD889e776bCBe2f0f9932EcFaBcDfCd1820
+* CIP37 mainnet: cfx:aceju902vhx1rtf8f6h3wp1p9m8r9xj2eau7fdwk48
+* CIP37 testnet: cfxtest:aceju902vhx1rtf8f6h3wp1p9m8r9xj2ea400xyd8y
+
+## create2 address
+* hex: 0x8a3a92281df6497105513b18543fd3b60c778e40
+* CIP37 mainnet: cfx:acfdzevjd15ew6jfme7vuzb94s5a276sjaxhcr9602 
+* CIP37 testnet: cfxtest:acfdzevjd15ew6jfme7vuzb94s5a276sja3pv730ww
+
+## ZERO address
+* hex: 0x0000000000000000000000000000000000000000
+* CIP37 mainnet: cfx:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa0sfbnjm2
+* CIP37 testnet: cfxtest:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa6f0vrcsw
+
+## Balance batch checker
```

Comment:
What is "Balance batch checker"? Is there any documentation about it?

Reply:
This an erc20 balance batch checker contract. Current i have remove it, seems it is not so important to document it here